hack/run-locally.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash 
+
+if [ -z "$CLUSTER_DIR" ]; then
```

Comment:
I'm not sure this ever all got written out anywhere, but preferred OpenShift shell-scripting style uses `[[`/`]]` and `${CLUSTER_DIR}`. (Likewise throughout.)